packages/google_ads/src/google_ads/GoogleAdsService.ts

```diff
@@ -1408,4 +1404,28 @@ export class GoogleAdsService {
       "login-customer-id": this.props.googleAds.accountId, // parent account id.
     };
   }
+
+  /**
+   * Google Auth Service.
+   *
+   * Request to reissue Google access token
+   */
+  private async refreshAccessToken(): Promise<string> {
+    const client = new google.auth.OAuth2(
```

Comment:
Add the googleapis dependency to package.json about all the google monorepo packages that use the `googleapis` package.